src/demo-app/demo-app/routes.ts

```diff
@@ -50,11 +50,13 @@ import {TypographyDemo} from '../typography/typography-demo';
 import {DemoApp, Home} from './demo-app';
 import {TableDemoPage} from '../table/table-demo-page';
 import {TABLE_DEMO_ROUTES} from '../table/routes';
+import { BadgeDemo } from 'badge/badge-demo';
```

Comment:
Spaces between braces should be omitted.

Reply:
Stupid editor.